packages/components/pie-modal/src/modal.scss

```diff
@@ -37,6 +37,15 @@
         outline: none;
     }
 
+    // We need to override the icon sizes at different screen sizes regardless of size prop passed in
+    pie-icon-button {
+        --btn-dimension: 40px;
+
+        @media (min-width: $breakpoint-wide) {
+            --btn-dimension: 48px;
+        }
```

Comment:
Could you make this a max-width query (for the 40px change), and then you wouldn't need to set it back to 48px at wide breakpoints?